src/node/attribute.rs

```diff
@@ -59,10 +59,17 @@ impl AttributeValueExpr {
     }
 }
 
+#[derive(Clone, Debug, syn_derive::ToTokens)]
+pub struct AttributeValueBlock {
+    pub token_eq: Token![=],
+    pub value: NodeBlock,
+}
+
 #[derive(Clone, Debug, syn_derive::ToTokens)]
 pub enum KeyedAttributeValue {
     Binding(FnBinding),
     Value(AttributeValueExpr),
+    Block(AttributeValueBlock),
```

Comment:
I'd recommend to move of handling invalid block into `AttributeValueExpr`, for two reasons:

1. currently `KeyedAttributeValue` contain completely different paths for parsing
2. currently if macro writer will get `KeyedAttributeValue::Value` he will know that it is value that user typed in `foo=""`